src/components/Section.astro

```diff
@@ -0,0 +1,41 @@
+---
+interface Props {
+    dark?: boolean;
+}
+
+const { dark = false } = Astro.props;
+---
+
+<section class={dark ? 'dark' : ''}>
+    <slot />
```

Comment:
**Suggestion:** Let's wrap this in an wrapper div that we can use to set the width incase we want to center content on larger screens. That can be another PR, and I'm happy to take that.